runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -527,10 +579,12 @@ public void onGameStateChanged(GameStateChanged gameStateChanged)
 		// Check for game state changes which removes a teleblock (Hopping or Login screen)
 		if (gameStateChanged.getGameState() == GameState.HOPPING || gameStateChanged.getGameState() == GameState.LOGIN_SCREEN)
 		{
-			removeGameTimer(HALFTB);
-			removeGameTimer(DMM_HALFTB);
-			removeGameTimer(FULLTB);
-			removeGameTimer(DMM_FULLTB);
+			removeTbTimers();
+		}
+
+		if (gameStateChanged.getGameState() == GameState.LOGGED_IN)
```

Comment:
else if (maybe you should change this entire method to do switch (gameStateChanged.getGameState()) instead, it will be cleaner